unicode.go

```diff
@@ -44,75 +36,32 @@ var (
 	doUnic = flag.Bool("u", false, "describe the characters from the Unicode database, in Unicode form")
 	doUNIC = flag.Bool("U", false, "describe the characters from the Unicode database, in glorious detail")
 	doGrep = flag.Bool("g", false, "grep for argument string in data")
-
-	doPrintUnicodeTxt = flag.Bool("db", false, "convert UnicodeData.txt to unicode.txt format on stdout")
 )
 
 var printRange = false
 
+//go:generate sh -c "curl http://ftp.unicode.org/Public/UNIDATA/UnicodeData.txt >UnicodeData.txt"
```

Comment:
this won't work on windows or plan 9 or another system without curl.

instead put this in the README, and the user can get it how they want.

Reply:
The user won't be running this - only you will, when you want to issue an update.

Reply:
ah of course.